ste/xferLogPolicy.go

```diff
@@ -18,6 +18,8 @@ import (
 // Because V10 SDK supports flexibility for injecting customized logging policy,
 // and considering redact x-amz-signature's request header for logging is not a general demand for Azure Storage Blob Go SDK.
 // TODO: Further discuss whether to add callback into RequestLogOptions for Azure Storage Blob Go SDK.
+// TODO: (new) consider also the relationship between the above comment and todos, and the new LogSanitizer
+//    Do we really need this copied version of the blob Storage SDK file now?
```

Comment:
I don't think we need this change copied over, but adding the constants over there may not be a bad idea either.

Reply:
The question was a TODO about whether we really need xferLogPolicy in AzCopy, not about whether we copy the recent changes to the SDK.  I don't think either of those questions is urgent. And both would be resolve if/when we one day rationalize the policy implementations.  So I propose to mark this resolved, with no further action. Lemme know if you think we should re-open it.